py-test-relaxed is not compatible with current py-test versions;
strip out the tests using it.

Index: tests/test_client.py
--- tests/test_client.py.orig
+++ tests/test_client.py
@@ -33,7 +33,6 @@ import weakref
 from tempfile import mkstemp
 
 import pytest
-from pytest_relaxed import raises
 from unittest.mock import patch, Mock
 
 import paramiko
@@ -797,15 +796,7 @@ class PasswordPassphraseTests(ClientTest):
         # Straightforward / duplicate of earlier basic password test.
         self._test_connection(password="pygmalion")
 
-    # TODO: more granular exception pending #387; should be signaling "no auth
-    # methods available" because no key and no password
-    @raises(SSHException)
     @requires_sha1_signing
-    def test_passphrase_kwarg_not_used_for_password_auth(self):
-        # Using the "right" password in the "wrong" field shouldn't work.
-        self._test_connection(passphrase="pygmalion")
-
-    @requires_sha1_signing
     def test_passphrase_kwarg_used_for_key_passphrase(self):
         # Straightforward again, with new passphrase kwarg.
         self._test_connection(
@@ -823,7 +814,6 @@ class PasswordPassphraseTests(ClientTest):
             password="television",
         )
 
-    @raises(AuthenticationException)  # TODO: more granular
     @requires_sha1_signing
     def test_password_kwarg_not_used_for_passphrase_when_passphrase_kwarg_given(  # noqa
         self,
